Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -94,9 +108,7 @@ final class CacheAdvanceTests: XCTestCase {
         try originalHeader.synchronizeHeaderData()
 
         let sut = try createCache(overwritesOldMessages: false, zeroOutExistingFile: false)
```

Comment:
We need to update the test name to reflect the new code.

Reply:
Good catch! Fixed in 073f1c6